packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -266,6 +266,14 @@ class Attributes {
     public setDefaultRepositoryAttributesIsDisabled() {
         return new CoreError("error.consumption.attributes.setDefaultRepositoryAttributesIsDisabled", "Setting default RepositoryAttributes is disabled for this Account.");
     }
+
+    public invalidTag(tag: string): ApplicationError {
+        return new ApplicationError("error.consumption.attributes.invalidTag", `The tag '${tag}' is invalid.`);
+    }
+
+    public invalidTags(tags: string[]): ApplicationError {
+        return new ApplicationError("error.consumption.attributes.invalidTags", `The tags '${tags.join(", ")}' is invalid.`);
```

Comment:
The current implementation would lead to the individual tags not each being enclosed with single ticks (`'tag1', 'tag2'`), but to all tags being enclosed together with one pair of single ticks (`'tag1, tag2'`), which is why I suggest the following change.

```suggestion
        return new ApplicationError("error.consumption.attributes.invalidTags", `The tags ${tags.map((tag) => `'${tag}'`).join(", ")} are invalid.`);
```